flang/lib/Semantics/check-omp-structure.cpp

```diff
@@ -2156,6 +2156,16 @@ void OmpStructureChecker::Leave(const parser::OmpClauseList &) {
         }
       }
     }
+
+    // 2.11.5 Simd construct restriction (OpenMP 5.1)
+    if (auto *clause{FindClause(llvm::omp::Clause::OMPC_safelen)}) {
+      if (FindClause(llvm::omp::Clause::OMPC_order)) {
```

Comment:
Presumably future standards might specify ordering other than concurrent. It would be best to check that here because it would be easy to forget it later.

Reply:
Done!